crates/starknet-devnet-core/src/starknet/mod.rs

```diff
@@ -767,7 +767,8 @@ impl Starknet {
         let raw_execution = RawExecution {
             calls: vec![Call {
                 to: erc20_address.into(),
-                selector: get_selector_from_name("mint").unwrap(),
+                selector: get_selector_from_name("mint")
+                    .map_err(|err| Error::UnexpectedInternalError { msg: err.to_string() })?,
```

Comment:
Why in same places we note that it is safe to use unwrap and in others, like this one, that it isn't?

Reply:
There are 2 places with `#[allow(clippy::unwrap/expect)]` the others are in the main file which it is ok if it fails, because it will terminate the program, which is desired effect and in some exported test methods, which to be used in another crates have to be included in the --release build. 
So what is left is StarknetConfig::default method and `get_unix_timestamp_as_seconds`. In `StarknetConfig` if `#[allow]` attribute is removed, then we will lose the trait implementation, because there is no default trait that returns Result.
`get_unix_timestamp_as_seconds` wont fail in the coming 584942417354 years.

Reply:
> get_unix_timestamp_as_seconds wont fail in the coming 584942417354 years.

Why not be ready when that moment comes? Perhaps we could create an issue so that we don't forget.

Reply:
Joking. Resolving.